UC NetCode 2024 Fall/Assets/App/Resource/Scripts/Obj/ProjectileObj.cs

```diff
@@ -0,0 +1,46 @@
+using ParrelSync.NonCore;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Pool;
+
+public class ProjectileObj : NetworkBehaviour
+{
+    [SerializeField] float _speed = 40f;
+    [SerializeField] private float _damage = 10;
+    [SerializeField] private float destructTime = 5f;
+
+    // TODO: Hook up projectile to pool system
```

Comment:
TODO: is just a placeholder, not a requirement for this assignment. I'm sorry that I confused you with this script